feature/news/src/main/java/kr/co/cotton/feature/news/NewsScreen.kt

```diff
@@ -69,39 +72,51 @@ fun NewsScreen(
 ) {
     val state = rememberLazyListState()
 
-    LazyColumn(
-        modifier = modifier,
-        state = state,
-        horizontalAlignment = Alignment.CenterHorizontally
+    CottonScaffold(
+        modifier = Modifier.fillMaxSize(),
+        topBar = {
+            CottonTopBar(
+                title = "News",
+                onClickBackBtn = { navController.popBackStack() }
+            )
+        }
     ) {
-        items(esportsNewsPagingList) { news ->
-            if (news != null) {
-                NewsCard(
-                    modifier = Modifier.padding(top = 16.dp, start = 16.dp, end = 16.dp),
-                    news = news,
-                    onClickCard = onClickItem
-                )
+        LazyColumn(
+            modifier = modifier,
+            state = state,
+            horizontalAlignment = Alignment.CenterHorizontally
+        ) {
+            items(esportsNewsPagingList) { news ->
+                if (news != null) {
+                    NewsCard(
+                        modifier = Modifier.padding(top = 16.dp, start = 16.dp, end = 16.dp),
+                        news = news,
+                        onClickCard = onClickItem
+                    )
+                }
             }
-        }
 
-        esportsNewsPagingList.apply {
-            when {
-                loadState.refresh is LoadState.Loading || loadState.append is LoadState.Loading -> {
-                    item {
-                        CottonLoadingView(
-                            modifier = Modifier
-                                .fillParentMaxSize()
-                                .padding(vertical = 16.dp)
-                        )
+            esportsNewsPagingList.apply {
+                when {
+                    loadState.refresh is LoadState.Loading
+                            || loadState.append is LoadState.Loading -> {
+                        item {
+                            CottonLoadingView(
+                                modifier = Modifier
+                                    .fillParentMaxSize()
+                                    .padding(vertical = 16.dp)
+                            )
+                        }
                     }
-                }
-                loadState.refresh is LoadState.Error || loadState.append is LoadState.Error -> {
-                    val e = esportsNewsPagingList.loadState.refresh as LoadState.Error
-                    item {
-                        Text(
-                            modifier = Modifier.fillParentMaxSize(),
-                            text = e.error.localizedMessage!!,
-                        )
+                    loadState.refresh is LoadState.Error
+                            || loadState.append is LoadState.Error -> {
```

Comment:
🚫 **[ktlint]** <chain-wrapping> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line must not begin with "||"